addresses/goerli.json

```diff
@@ -1,185 +1,185 @@
-					"fee": "0.1",
-					"timeStretch": 27.7
-				},
-				"ytPool": {
-					"address": "0x4b7C8509Ee69D9131c0923E4F00d0D928594FF47",
-					"poolId": "0x4b7c8509ee69d9131c0923e4f00d0d928594ff470002000000000000000000a6",
-					"fee": "0.003"
-				},
-				"weightedPoolFactory": "0x7576dB443020ec4F46526c317262347Acdf1B99d",
-				"convergentCurvePoolFactory": "0x0766B218517d9dC198155f0dC3485270cF788aF7"
-			}
-		],
-		"weth": [
-			{
-				"expiration": 1628210937,
-				"address": "0x89d66Ad25F3A723D606B78170366d8da9870A879",
-				"ptPool": {
-					"address": "0x9eB7F54C0eCc4d0D2dfF28a1276e36d598F2B0D1",
-					"poolId": "0x9eb7f54c0ecc4d0d2dff28a1276e36d598f2b0d1000200000000000000000066",
-					"fee": "0.1",
-					"timeStretch": 8
-				},
-				"ytPool": {
-					"address": "0xD75bfF2444FF738d443066ff4688691e6852b217",
-					"poolId": "0xd75bff2444ff738d443066ff4688691e6852b217000200000000000000000067",
-					"fee": "0.1"
-				},
-				"trancheFactory": "0x5690332C2f0c12F00c147cE350d95B19a0C24f14",
-				"weightedPoolFactory": "0x7576dB443020ec4F46526c317262347Acdf1B99d",
-				"convergentCurvePoolFactory": "0x0766B218517d9dC198155f0dC3485270cF788aF7"
-			},
-			{
-				"expiration": 1621010937,
-				"address": "0x44eecA004b2612d131EDA7dA2b9d986E7fED562e",
-				"ptPool": {
-					"address": "0x5941DB4d6C500C4FFa57c359eE0C55c6b41D0b61",
-					"poolId": "0x5941db4d6c500c4ffa57c359ee0c55c6b41d0b61000200000000000000000064",
-					"fee": "0.1",
-					"timeStretch": 8
-				},
-				"ytPool": {
-					"address": "0xcF6894C48c2AF3ddD433CC1EDfEfC74e654cC9B4",
-					"poolId": "0xcf6894c48c2af3ddd433cc1edfefc74e654cc9b4000200000000000000000065",
-					"fee": "0.1"
-				},
-				"trancheFactory": "0x5690332C2f0c12F00c147cE350d95B19a0C24f14",
-				"weightedPoolFactory": "0x7576dB443020ec4F46526c317262347Acdf1B99d",
-				"convergentCurvePoolFactory": "0x0766B218517d9dC198155f0dC3485270cF788aF7"
-			}
-		],
-		"dai": [
-			{
-				"expiration": 1624901889,
-				"address": "0x6866dFc9A60e9dba922668b9b27931DCaCDF645A",
-				"trancheFactory": "0x5690332C2f0c12F00c147cE350d95B19a0C24f14",
-				"weightedPoolFactory": "0x7576dB443020ec4F46526c317262347Acdf1B99d",
-				"convergentCurvePoolFactory": "0x0766B218517d9dC198155f0dC3485270cF788aF7",
-				"ptPool": {
-					"address": "0x00655F8F287848f35538648A012e2cbd13016138",
-					"poolId": "0x00655f8f287848f35538648a012e2cbd13016138000200000000000000000079",
-					"fee": "0.1",
-					"timeStretch": 26
-				},
-				"ytPool": {
-					"address": "0x4c3f56cC1ad2BC61293739C7f0550575c9DAf313",
-					"poolId": "0x4c3f56cc1ad2bc61293739c7f0550575c9daf313000200000000000000000078",
-					"fee": "0.003"
-				}
-			}
-		]
-	},
-	"wrappedCoveredPrincipalTokenFactory": "0x7E61D33C76C781821138B0972e25b84c4C0150Eb",
-	"wrappedCoveredPrincipalToken": {
-		"dai": "0x0c5383dffbD66c07820DD4278EE3f0CaBa08ED22"
-	}
-}
\ No newline at end of file
+  "balancerVault": "0x65748E8287Ce4B9E6D83EE853431958851550311",
```

Comment:
I think the changes to this file are just the tab-type format changes, we've talked about these in previous PR's but would prefer we keep those separate from the address changes

Reply:
Yeah they are mostly format changes, my editor does it automatically, which I should probably take off.  This'll be annoying to undo so I'll add a comment in the PR description.